### PR TITLE
Add enrichments for Tweet URL and user profile URL

### PR DESCRIPTION
**Pull request by psavkar**

### CODE REVIEW

File: components/twitter/sources/my-liked-tweets/my-liked-tweets.js [hunk 1]
```diff
@@ -17,7 +17,7 @@ module.exports = {
   dedupe: "unique",
   async run(event) {
     (await this.twitter.getLikedTweets()).reverse().forEach(tweet => {
-      this.$emit(tweet, {
+      this.$emit(this.twitter.enrichTweets(tweet), {
```
Comment (tjk) [inline components/twitter/sources/my-liked-tweets/my-liked-tweets.js:20]: `enrichTweet` no?

File: components/twitter/sources/tweets-liked-by-user/tweets-liked-by-user.js [hunk 2]
```diff
@@ -18,7 +19,7 @@ module.exports = {
   dedupe: "unique",
   async run(event) {
     (await this.twitter.getLikedTweets({ screen_name: this.screen_name })).reverse().forEach(tweet => {
-      this.$emit(tweet, {
+      this.$emit(this.twitter.enrichTweets(tweet), {
```
Comment (tjk) [inline components/twitter/sources/tweets-liked-by-user/tweets-liked-by-user.js:22]: same here

File: components/twitter/twitter.app.js [hunk 3]
```diff
@@ -225,6 +225,18 @@ module.exports = {
...
+    /**
+    * Enrich a Tweet object with ISO 8601 and timestamp (in milliseconds) representations of the `created_at` date/time, Tweet URL and user profile URL.
+    * @params {Object} tweet - An object representing a single Tweet as returned by Twitter's API
+    * @returns {Object} An enriched Tweet object is returned.
+    */
+    enrichTweet(tweet) {
+      tweet.created_at_timestamp = moment(tweet.created_at, 'ddd MMM DD HH:mm:ss Z YYYY').valueOf()
+      tweet.created_at_iso8601 = moment(tweet.created_at, 'ddd MMM DD HH:mm:ss Z YYYY').toISOString()
+      tweet.url = `https://twitter.com/${tweet.user.screen_name}/statuses/${tweet.id_str}`
+      if(tweet.user) { tweet.user.profile_url = `https://twitter.com/${tweet.user.screen_name}/` }
```
Comment (tjk) [inline components/twitter/twitter.app.js:237]: we should set up a linter in this repo, but until then, just try to mimic the style in the rest of the code... space after `if` and if branch is on same line as conditional, don't use braces, or keep braces but just indent the branch in a new line.

File: components/twitter/sources/my-liked-tweets/my-liked-tweets.js (hunk 1 above)
Comment (psavkar) [reply components/twitter/sources/my-liked-tweets/my-liked-tweets.js:20]: omg. yes. fixing

**Review by jverce — APPROVED** [review]

**Pull request merged**